src/main/scala-sbt-0.13/compat.scala

```diff
@@ -4,12 +4,21 @@ package sbt
 // Exposes (a minimal part) of the sbt 1 API using sbt 0.13 API
 // Inspired by macro-compat
 
+import scala.language.implicitConversions
+
 object compat {
   val scalaModuleInfo = SettingKey[Option[librarymanagement.ScalaModuleInfo]]("ivyScala")
 
   implicit class ModuleIdOps(val _m: ModuleID) extends AnyVal {
     def withName(n: String): ModuleID = _m copy (name = n)
   }
+
+  implicit class RichProcessBuilder(pb: ProcessBuilder) {
+    def lineStream_!(logger: ProcessLogger) = pb.lines_!(logger)
+    def lineStream_! : Stream[String]       = pb.lines_!
+  }
+
+  implicit def absLog2PLog(log: AbstractLogger): ProcessLogger = sbt.Logger.absLog2PLog(log)
```

Comment:
My response is in a collapsed thread, with the new commit, so reposting here:

> however, you need to import sbt.Logger._. I think it would make sense to make it available here as well. This way, you only need to import sbt.compat._ and it solves several issues with one import.

Are you sure? Unless I'm missing something by being defined on the companion object it's in implicit scope, for both sbt 0.13 and sbt 1, so it's ok?

Btw I'm happy to find a way to include this is need be, I'm just not convinced of the need.

Reply:
from scala-native:

import scala.sys.process._

```
class ComRunner(bin: File,
                envVars: Map[String, String],
                args: Seq[String],
                logger: Logger) {

  private[this] val runner = new Thread {
    override def run(): Unit = {
      val port = serverSocket.getLocalPort
      logger.info(s"Starting process '$bin' on port '$port'.")
      Process(bin.toString +: port.toString +: args, None, envVars.toSeq: _*) ! logger
    }
  }
```
```
[error]  found   : sbt.Logger
[error]  required: scala.sys.process.ProcessLogger
[error]       Process(bin.toString +: port.toString +: args, None, envVars.toSeq: _*) ! logger
```

Reply:
Ok now I see what the use case is. So you want to use scala.sys.process's Process with sbt's Logger.

To do that for both sbt 0.13 and sbt 1 code I think you'd need implicit conversions for both.